src/Middleware/Diagnostics.Abstractions/src/IExceptionHandlerEndpointFeature.cs

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Routing;
+
+namespace Microsoft.AspNetCore.Diagnostics
+{
+    /// <summary>
+    /// Represents an exception handler with the original Endpoint and path of the request.
+    /// </summary>
+    public interface IExceptionHandlerEndpointFeature : IExceptionHandlerPathFeature
```

Comment:
Is the derivation necessary or useful here? We did it before as a way of expanding the base feature, but that made more sense when the values for IExceptionHandlerPathFeature were always set. IExceptionHandlerEndpointFeature 
 is optional so let's put both interfaces on ExceptionHandlerFeature instead.

Reply:
IExceptionHandlerPathFeature did it. I don't think there is any value but might as well keep the consistent pattern, at least for these exception handler features.